### PR TITLE
fix: initial version of this repo

### PR DESCRIPTION
**Pull request by dgschwindturo**

### CODE REVIEW

**Comment by Toolo** [comment]
+1 to @shakefu comments 😄 

Let's convert to native eslint/pre-commit, update the publishing target and 🚀

**Review by Toolo — APPROVED** [review]
🚀

**Comment by dgschwindturo** [comment]
Yep, agree, the Test workflow is not needed. Now, for me to figure out where that needs to be culled.

I have added a .pre-commit-config.yaml file, but sorting through why the Lint workflow fails.

**Comment by dgschwindturo** [comment]
The extra required status check of `Test` has been culled.

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Release
+on:
+  push:
+    branches: [main]
+jobs:
+  release:
```
Comment (greenkiwi) [inline .github/workflows/release.yaml:6]: This should probably have `lint` as a job here that is `needed` by release.

Comment (dgschwindturo) [reply .github/workflows/release.yaml:20]: Ok, very well, I have added the `Lint` job as well as Release needing it.

**Review by Toolo — APPROVED** [review]

**Pull request merged**